lib/osf-components/addon/components/node-publication-doi-editable/component.ts

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import { PublicationDoiManager } from 'osf-components/components/editable-field/publication-doi-manager/component';
+import styles from './styles';
+import template from './template';
+
+@layout(template, styles)
+export default class NodePublicationDoiEditable extends Component {
+    // Private
+    manager!: PublicationDoiManager;
+
+    doiPlaceholder = '10.xxxx/xxxxx';
+    doiUrlPrefix = 'https://doi.org/';
```

Comment:
These two would make sense as constants in `utils/extract-doi` (maybe also rename it to `utils/doi`, so we can put all DOI-related utils together)